src/c2goto/cprover_library.cpp

```diff
@@ -6,11 +6,15 @@ Author: Daniel Kroening, kroening@kroening.com
 
 \*******************************************************************/
 
+#include <ac_config.h>
 #include <c2goto/cprover_library.h>
 #include <cstdlib>
+#include <fstream>
 #include <goto-programs/read_goto_binary.h>
 #include <util/c_link.h>
 #include <util/config.h>
+#include <util/language.h>
+#include <boost/filesystem.hpp>
```

Comment:
Shouldn't this include come earlier? I believe we use alphabetical order, but we need to double-check it.

Reply:
Oh, alphabetical! I was wondering... Will fix it.

Reply:
Done